4.1/robot/Dockerfile

```diff
@@ -21,8 +21,8 @@ RUN pip3 install --upgrade --no-cache-dir setuptools pip virtualenv
 # version definitions
 ARG PYATS_VERSION=4.1.0
 ARG PYATS_ROBOT_VERSION=4.1.1
-ARG GENIE_VERSION=3.0.0
-ARG GENIE_ROBOT_VERSION=3.0.7
```

Comment:
This was a bit confusing to me (at first) as I was not accounting for all the other genie packages.